sources/packages/backend/libs/integrations/src/sfas-integration/sfas-integration.module.ts

```diff
@@ -21,21 +21,21 @@ import {
     SshService,
     SFASIntegrationService,
     SFASIntegrationProcessingService,
-    SFASIndividualService,
-    SFASApplicationService,
-    SFASRestrictionService,
-    SFASPartTimeApplicationsService,
+    SFASIndividualImportService,
+    SFASApplicationImportService,
+    SFASRestrictionImportService,
+    SFASPartTimeApplicationsImportService,
     SIMSToSFASService,
     SIMSToSFASIntegrationService,
     SIMSToSFASProcessingService,
   ],
   exports: [
     SFASIntegrationService,
     SFASIntegrationProcessingService,
-    SFASIndividualService,
-    SFASApplicationService,
-    SFASRestrictionService,
-    SFASPartTimeApplicationsService,
+    SFASIndividualImportService,
+    SFASApplicationImportService,
+    SFASRestrictionImportService,
+    SFASPartTimeApplicationsImportService,
```

Comment:
Is there a need to export the below services?
```
SFASIndividualImportService,
SFASApplicationImportService,
SFASRestrictionImportService,
SFASPartTimeApplicationsImportService,
```
The service being consumed externally is probably `SFASIntegrationProcessingService` and this would be the only one that needs to be exported. Please give it a try.

Reply:
make sense, changed now